variables.tf

```diff
@@ -85,6 +85,19 @@ variable "vpc_security_groups" {
   default     = null
 }
 
+variable "assume_role_policies" {
```

Comment:
This could better be renamed `additional_assume_role_policies` maybe. What do you think?

Reply:
yeah, additional is probably neater, I don't mind either way though

Reply:
description is clear anyway, so the docs should explain it